seedfarmer/__init__.py

```diff
@@ -79,6 +80,11 @@ def _load_config_data(self) -> None:
             config_data: Dict[str, Any] = yaml.safe_load(file)
             self._PROJECT = config_data["project"]
             self._DESCRIPTION = config_data["description"] if "description" in config_data else "NEW PROJECT"
+            self._PROJECT_POLICY_PATH = (
+                os.path.join(self._OPS_ROOT, config_data["projectPolicyPath"])
```

Comment:
as discussed, let's create a housekeeping ticket to use a dataclass and a schema for this. approving here

Reply:
Complete